openshift-hack/e2e/namespace.go

```diff
@@ -33,6 +33,9 @@ func CreateTestingNS(ctx context.Context, baseName string, c kclientset.Interfac
 			labels = map[string]string{}
 		}
 		labels["security.openshift.io/disable-securitycontextconstraints"] = "true"
+		// turn off the OpenShift label syncer so that it does not attempt to sync
```

Comment:
when is "isKubeNamespace" false?

Reply:
In origin we have it set here: https://github.com/openshift/origin/blob/master/test/extended/util/test_setup.go#L89-L91, in k8s it's always true https://github.com/openshift/kubernetes/blob/master/openshift-hack/cmd/k8s-tests/provider.go#L103-L105